docker/1.11.0/Dockerfile.cpu

```diff
@@ -55,6 +57,7 @@ COPY $framework_support_installable .
 
 RUN pip install --no-cache-dir -U \
     keras==2.2.4 \
+    retrying \
```

Comment:
sagemaker containers installs retrying, not the containers. You can remove this statement.

Reply:
done